iphone/Classes/DeMarcelpociotCollectionviewCollectionView.h

```diff
@@ -8,7 +8,6 @@
 #import "TiUIView.h"
 #import "DeMarcelpociotCollectionviewCollectionViewProxy.h"
 #import "CHTCollectionViewWaterfallLayout.h"
-#import "M13ContextMenu.h"
```

Comment:
I agree, but I think people would rather want a stable, fast collection-view module than possible crashes with external libraries. The easiest thing would be to make an own module for the context-menu and make t able to link to the collection-view, but also to any other view as well. 
I do not plan to move people to the fork, you how I work. So if you want it restored, you can still cherry-pick the relevant changes and keep the external library in.
The module looks like a straight copy of the Ti.UI.ListView (even same property names like "tableView", but a bit outdated, so we should compare the latest list-view changes to check the possible main-thread issues. I feel like Apple will deprecate UITableView in iOS 11, so this might get a priority bump soon.